README.md

```diff
@@ -3,10 +3,13 @@ KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.
 
 A KFService encapsulates the complexity of autoscaling, networking, health checking, server configuration, and more, to provide customers with a simple and seamless experience when deploying models.
 
-In the future, we hope to support more advanced use cases such as outlier detection, bias detection, explainability, pre/post processing, and performance profiling across infrastructure configurations.
+This project is an evolution of the [original proposal in the Kubeflow repo](https://github.com/kubeflow/kubeflow/issues/2306). 
```

Comment:
Don`t want to list future roadmap?

Reply:
I think it's the type of thing that gets stale. I've cleaned up the language a little bit to be more "present", but it's describe as a "story" instead of "current reality" :)